### PR TITLE
(changes made) Add More PT-BR courses

### PR DESCRIPTION
**Pull request by rubenmarcus**
## What does this PR do?
Add resource(s) 

### Description
This P/R adds more free programming courses in PT-BR language

### Why is this valuable (or not)?
- it updates the list with more recently launched courses

### How do we know it's really free?
- The courses are openly available on Youtube

### For book lists, is it a book? For course lists, is it a course? etc.
- provides information regarding several languages / technologies

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
when you've made changes, please add `(changes made)` to the title of your PR

**Comment by rubenmarcus** [comment]
@eshellman I see but some of the videos arent playlists they full courses in only 1 video

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -297,6 +311,7 @@
 ### Programação
 
 * [Algoritmos de Ordenação](https://www.youtube.com/playlist?list=PLzZut2slkqywtFxqTY8AQwIG65h_2oMBL) - Bruno Ribas
+* [Curso Lógica de Programação Completo 2023 [Iniciantes] + Desafios + Muita prática](https://www.youtube.com/watch?v=iF2MdbrTiBM) - Jonathan de Souza
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:314]: 3 hours. ok.

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -273,6 +286,7 @@
 * [Curso de Node.js](https://www.youtube.com/playlist?list=PLJ_KhUnlXUPtbtLwaxxUxHqvcNQndmI4B) - Victor Lima Guia do Programador
 * [Do Zero A Produção: Aprenda A Construir Uma API Node.Js Com Typescript](https://www.youtube.com/playlist?list=PLz_YTBuxtxt6_Zf1h-qzNsvVt46H8ziKh) - Waldemar Neto Dev Lab
 * [Imersão em desenvolvimento de APIs com Node.js](https://erickwendel.teachable.com/p/node-js-para-iniciantes-nodebr) - Erick Wendel (Teachable)
+* [Nest.js do zero aos microsserviços](https://www.youtube.com/watch?v=VP5-RKpfHIg) - Full Cycle
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:289]: what? you have to subscribe to see the course. declined.

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -201,11 +203,22 @@
 
 ### JavaScript
 
-* [Bootcamp da Brainn de React](https://www.youtube.com/playlist?list=PLF7Mi9HNzvVmzOyDyl--xQVdi60jxduU1) - Canal Brainn Co.
 * [Curso de Javascript Completo](https://www.youtube.com/playlist?list=PL2Fdisxwzt_d590u3uad46W-kHA0PTjjw) - Programação Web
 * [Curso Grátis de JavaScript e ECMAScript para Iniciantes](https://www.youtube.com/playlist?list=PLHz_AreHm4dlsK3Nr9GVvXCbpQyHQl1o1) - Gustavo Guanabara (Curso em Vídeo)
-* [Curso Intensivo de Next.js & React](https://www.cod3r.com.br/courses/curso-intensivo-next-react) - Leonardo Leitão (Cod3r)
+* [Curso Javascript Completo 2023 [Iniciantes] + 14 Mini-Projetos](https://www.youtube.com/watch?v=i6Oi-YtXnAU) - Jhonatan de Souza
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:208]: 4 hours. ok

File: courses/free-courses-pt_BR.md [hunk 4]
```diff
@@ -201,11 +203,22 @@
...
 * [Fast & Furious](https://www.youtube.com/playlist?list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts
+* [Kubernetes para Devs Javascript](https://www.youtube.com/watch?v=Co1xmcQS6Bw&list=PLqFwRPueWb5ccEFVx5vOvrKlYT_uQ3aQw) - Erick Wendel e Lucas Santos
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:210]: use the playlist

**Review by eshellman — CHANGES_REQUESTED** [review]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
we'll accept single videos > 1 hr if there's no playlist
when you've made changes, please add `(changes made)` to the title of your PR

**Comment by rubenmarcus** [comment]
thanks @eshellman  done the changes

File: courses/free-courses-pt_BR.md [hunk 5]
```diff
@@ -201,11 +203,22 @@
...
+* [Testes no NodeJS aplicando TDD com Jest](https://www.youtube.com/watch?v=2G_mWfG0DZE) - Diego Fernandes
+
+
+#### React
+
+* [Aprenda Next.js,GraphQL e Leaflet na prática!](https://www.youtube.com/watch?v=WOqF_pSzyvQ&list=PLlAbYrWSYTiPlXj6USip_lCPzONUATJbE&pp=iAQB) - Willian Justen
+* [Bootcamp da Brainn de React](https://www.youtube.com/playlist?list=PLF7Mi9HNzvVmzOyDyl--xQVdi60jxduU1) - Canal Brainn Co.
+* [Curso Intensivo de Next.js & React](https://www.cod3r.com.br/courses/curso-intensivo-next-react) - Leonardo Leitão (Cod3r)
+* [Curso Intensivo de React e Next 13](https://www.youtube.com/playlist?list=PLdPPE0hUkt0q_XT8zDe313e27wFQ-k9fe) - Cod3r Cursos
+* [Next js 13.4 Masterclass Prático c/ Stripe e Shadcn-ui](https://www.youtube.com/watch?v=b0AinejyevM&list=PLR8OzKI52ppWoTRvAmB_FQPPlHS0otV7V&index=19) - DeveloperDeck101
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:221]: I think you want https://www.youtube.com/playlist?list=PLR8OzKI52ppWoTRvAmB_FQPPlHS0otV7V

**Review by eshellman — CHANGES_REQUESTED** [review]
a few more changes

**Comment by rubenmarcus** [comment]
@eshellman  thanks for the patience, and also for the detailed review! Done the suggested changes!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**